docs/introduction.md

```diff
@@ -4,145 +4,62 @@
 
 ### Platform agnostic developer tools
 
-The typical example is [Waypoint](https://github.com/hashicorp/waypoint). It is
-a user facing tool which introduces a consistent workflow (i.e., build,
-deploy, release) for developers to ship applications on top of different platforms.
-KubeVela can be integrated into Waypoint like any other supported platforms. 
-In this case, developers can use the Waypoint workflow instead of the KubeVela
-CLI to manage applications.
+The typical example is [Waypoint](https://github.com/hashicorp/waypoint). Waypoint is a developer facing tool which introduces a consistent workflow (i.e., build, deploy, release) to ship applications on top of different platforms.
+
+KubeVela can be integrated into Waypoint like any other supported platforms. In this case, developers will use the Waypoint workflow instead of the KubeVela Appfile/CLI to manage applications, and all the capabilities of KubeVela including abstractions will still be available in this integration.
 
+### Helm, etc. 
 
-### Package management tools 
+Helm is a package manager for Kubernetes that provides package, install, and upgrade a set of YAML files for Kubernetes as a unit. 
 
-People may mistakenly think KubeVela is another package manager like Helm.
-Although using Helm chart significantly reduces the burden of managing a
-complicated Kubernetes application, whoever prepares the helm chart cannot avoid
-the tedious and error-prone work of packaging those YAML files.
+KubeVela is not a package manager. Developers are expected to use KubeVela's Appfile to describe how to build and deploy application and that Appfile is not Kubernetes YAML format. However, the server side API object behind Appfile (i.e. `Application Configuration`) is indeed a Kubernetes CRD, it could be packaged and distributed by Helm at ease.
 
-KubeVela aims to fundamentally remove the need of managing conventional Kubernetes
-YAML files. However, in the server side, KubeVela still relies on Helm to package
-and manage the third-party plugins such as `Prometheus`, etc.
+In implementation side, KubeVela heavily relies on Helm to package and manage its third-party plug-ins such as `Prometheus`, etc. 
 
 ### Kubernetes
 
-KubeVela can be treated as a special Kubernetes "distribution" that focuses on
-developers and application management only.
-It leverages the native Kubernetes extensibility to resolve a hard problem - making
-application management enjoyable in Kubernetes.
+KubeVela is a Kubernetes addon. Though for platform builders, it's reasonable to think KubeVela as a special Kubernetes "distribution" that focuses on developers and application management only. It leverages the native Kubernetes extensibility and capabilities to resolve a hard problem - making application management enjoyable on Kubernetes.
```

Comment:
How about just say 
`KubeVela is a special Kubernetes "distribution" that focuses on developers and application management only`

We probably need a clear msg here so saying addon, platform builders etcs make the claim unclear.